e2e/dashboard.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { test, expect, Page } from "@playwright/test"
+
+import {
+  E2E_EMAIL_REPO_MASTER_LINK,
+  E2E_EMAIL_REPO_STAGING_LINK,
+  E2E_EMAIL_TEST_SITE,
+  ISOMER_GUIDE_LINK,
+} from "./fixtures/constants"
+import { setEmailSessionDefaults } from "./utils/session"
+
+const REVIEW_MODAL_SUBTITLE =
+  "An Admin needs to review and approve your changes before they can be published"
+
+const REVIEW_REQUEST_ALERT_MESSAGE =
+  "There’s a Review request pending approval. Any changes you make now will be added to the existing Review request, and published with the changes in it."
+
+const getReviewRequestButton = async (page: Page) => {
```

Comment:
nit: I think this doesn't need to be async?

Reply:
updated